### PR TITLE
Attempt to update Travis yml now that java8 is EOL

### PR DESCRIPTION
**Pull request by anne-decusatis**
https://docs.travis-ci.com/user/languages/java/#switching-jdks-java-8-and-below-within-one-job

I have no idea if this is valid or how to test it but it's already not working so my suggestion is we test in prod...

### CODE REVIEW

**Comment by nevillelyh** [comment]
I just switched all my builds to `openjdk8`. Simpler and should be close enough to Oracle JDK.

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,13 @@
...
+addons:
+  apt:
+    packages:
+      - openjdk-8-jre
+
 language: scala
 scala:
   - 2.11.12
   - 2.12.6
-jdk: oraclejdk8
```
Comment (nevillelyh) [inline .travis.yml]: Just replacing this with `openjdk8` should work?
https://github.com/nevillelyh/shapeless-datatype/blob/master/.travis.yml#L7

Comment (anne-decusatis) [reply .travis.yml]: thank you!! really appreciate the example @nevillelyh ! https://github.com/spotify/ratatool/pull/164 is looking like it's passing so I will close this in favor of that

**Comment by anne-decusatis** [comment]
closing in favor of https://github.com/spotify/ratatool/pull/164

**Pull request closed without merging**